lncfg/sweeper.go

```diff
@@ -16,5 +19,11 @@ func (s *Sweeper) Validate() error {
 		return fmt.Errorf("batchwindowduration must be positive")
 	}
 
+	// We require the max fee rate to be at least 100 sat/vbyte.
+	if s.MaxFeeRate < chainfee.AbsoluteFeePerKwFloor*1e2 {
+		return fmt.Errorf("maxfeerate must be greater than " +
+			"25,000 sat/kw(100 sat/vb)")
+	}
```

Comment:
also check if `s.MaxFeeRate > DefaultMaxFeeRate`?

Reply:
hmm what if it's sweeping a mega large HTLC? What's the max allowed value you have in mind?

Reply:
I think it's more likely that somebody fat-fingers this and adds a 0, but I can do the math and figure out something reasonable

Reply:
cool capped at 10_000